--- ../src/mirage_logs.ml	2019-07-12 18:55:31.244619909 +0530
+++ ../src/mirage_logs_new.ml	2019-07-12 18:59:33.638897391 +0530
@@ -97,7 +97,7 @@
         Buffer.clear buf;
         if level <= console_threshold src then
           Printf.fprintf ch "%s: %s\n%!" (fmt_timestamp (posix_time, tz)) msg;
-        MProf.Trace.label msg;
+        (* MProf.Trace.label msg; *)
         log_to_ring posix_time tz msg ring;
         over ();
         k () in
